.github/workflows/unittests.yaml

```diff
@@ -28,21 +31,13 @@ jobs:
           fetch-depth: 0
           lfs: true
       - name: Checkout LFS objects
-        run: git lfs checkout        
-      - name: Setup Scala
-        uses: olafurpg/setup-scala@v12
-        with:
-          java-version: adopt-openj9@1.8.0-292
-      - name: Cache .ivy2
-        uses: actions/cache@v4
-        with:
-          path: ~/.ivy2
-          key: ${{ runner.os }}-ivy2-${{ hashFiles('**/build.sbt') }}
-      - name: Cache .sbt
-        uses: actions/cache@v4
+        run: git lfs checkout
+      - uses: actions/setup-java@v4
         with:
-          path: ~/.sbt
-          key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.properties') }}
+          distribution: 'temurin'
+          java-version: ${{ matrix.java-version }}
+          cache: sbt
+      - uses: sbt/setup-sbt@v1
       - name: Unit Tests
```

Comment:
@nh13 do you think we can delete lines 46 and 47?

Do you recall what they are for?

Reply:
Those files may cause unnecessary cache misses, causing us to not use the cache: https://www.scala-sbt.org/1.x/docs/Travis-CI-with-sbt.html#Caching

Reply:
Then I do think I can delete them since the GitHub Action to cache is smart and only looks for certain keys, instead of requiring we delete stuff that will dirty the cache.

https://github.com/actions/setup-java/blob/f4f1212c880fdec8162ea9a6493f4495191887b4/README.md?plain=1#L128

Thanks!

Reply:
I thinks was also a hold over from using travis-ci